mac-creator.sh

```diff
@@ -0,0 +1,17 @@
+# cleaning Mac's build folder
```

Comment:
I think the installer specific scripts should be in a subdirectory. I was hoping that they could be written using node scripts; did you look at using `electron-osx-sign` instead of calling `codesign` directly?